custom/icds_reports/static/js/directives/registered-household/registered-household.directive.js

```diff
@@ -14,7 +14,7 @@ function RegisteredHouseholdController($scope, $routeParams, $location, $filter,
     vm.data = {
         legendTitle: 'Total AWCs that have launched ICDS CAS',
     };
-    vm.filters = ['age', 'gender'];
+    vm.filters = ['age', 'gender', 'ageServiceDeliveryDashboard'];
```

Comment:
Why does this need to be added everywhere?

Reply:
> besides overwriting its own files overwrites all report directives as vm.filters isn't a list of used additional filters in the report but of not used.
Adding it everywhere disables it everywhere else.